docs/completion.md

```diff
@@ -31,18 +31,13 @@ Place the completion scripts in `/etc/bash_completion.d/` (`` `brew --prefix`/et
 
 Completion will be available upon next login.
 
-<!--
-### Zsh
 
-**Note: there does not seem to be any Zsh completion file yet, but when there is, documentation could look like this.**
+### Zsh
 
 Place the completion scripts in your `/path/to/zsh/completion`, using e.g. `~/.zsh/completion/`
 
     mkdir -p ~/.zsh/completion
-    files=(docker-machine docker-machine-wrapper docker-machine-prompt)
-    for f in "${files[@]}"; do
-      curl -L https://raw.githubusercontent.com/docker/machine/v$(docker-machine --version | tr -ds ',' ' ' | awk 'NR==1{print $(3)}')/contrib/completion/zsh/$f > ~/.zsh/completion/_$f
-    done
+    curl -L https://raw.githubusercontent.com/docker/machine/v$(docker-machine --version | tr -ds ',' ' ' | awk 'NR==1{print $(3)}')/contrib/completion/zsh/_docker-machine > ~/.zsh/completion/_docker-machine
```

Comment:
these aren't in the current release yet https://github.com/docker/machine/tree/v0.7.0/contrib/completion so probably needs to be removed from the cherry picks